src/eterna/ui/toolbar/Toolbar.ts

```diff
@@ -530,6 +532,7 @@ export default class Toolbar extends ContainerObject {
         this.magicGlueButton = this.setupButton(magicGlueButtonProps, this._showGlue);
         this.stampTLoopA = this.setupButton(stampTLoopAMenuButtonProps, this._showStampTLoop);
         this.stampTLoopB = this.setupButton(stampTLoopBMenuButtonProps, this._showStampTLoop);
+        this.autoSolverButton = this.setupButton(AutoSolverMenuButtonProps, true);
```

Comment:
This should be gated by a feature flag. Also for the purposes of this study, maybe we want to automatically place it into the hotbar on initialization if enabled? (Not (just) a default tool, as players should necessarily have saved settings from playing without it present)